Lib/test/libregrtest/main.py

```diff
@@ -100,6 +101,13 @@ def __init__(self):
         self.next_single_test = None
         self.next_single_filename = None
 
+        self.getloadavg = None
+        if hasattr(os, 'getloadavg'):
+            self.getloadavg = lambda: os.getloadavg()[0]
+        elif sys.platform == 'win32':
+            self.load_tracker = WindowsLoadTracker()
+            self.getloadavg = lambda: self.load_tracker.getloadavg()
```

Comment:
This can just be `self.getloadavg = self.load_tracker.getloadavg`

Reply:
And since there are no later references to `self.load_tracker`, that doesn't need to be kept on the object (the references via the method will keep it from being deallocated)

Reply:
Done, thanks :)